source/installation/index.html

```diff
@@ -9,29 +9,29 @@
 
 <div class="intro">
     <p>
-        Home Assistant offers four different installation methods. We recommend one of the following two:
+        Home Assistant offers four different installation methods. We recommend using one of the following two methods:
     </p>
     <ul>
         <li><b>Home Assistant Operating System</b>: Minimal Operating System optimized to power Home Assistant. 
-        Comes with Supervisor to manage Home Assistant Core and add-ons pre-installed. Recommended installation type.</li>
+        It comes with Supervisor to manage Home Assistant Core and pre-installed add-ons. Recommended installation method.</li>
         <li><b>Home Assistant Container</b>: Standalone container-based installation of Home Assistant Core (e.g. Docker).</li>
     </ul>
-    <p>In addition there are two installation methods available for experienced users:</p>
+    <p>There are two alternative installation methods available for experienced users:</p>
     <ul>
         <li><b>Home Assistant Supervised</b>: Manual installation of the Supervisor.</li>
-        <li><b>Home Assistant Core</b>: Manual installation using a Python virtual environment.</li>
+        <li><b>Home Assistant Core</b>: Manual installation using Python virtual environment.</li>
     </ul>
     <p>
-        The list below shows which installation method is available on what device or platform.
+        The list below shows the compatibility of the above installation methods with the different devices and platforms.
```

Comment:
It isn't aimed at compatibility, I think that is misleading in this new text.

Reply:
'The list below shows the compatibility of the above installation methods with the different device or platform' can be modified as 'The list below shows the installation method available based on the device and platform being used'. Please let me know if this is better.